pkg/agent/apiserver/apiserver.go

```diff
@@ -76,14 +79,28 @@ func New(aq agentquerier.AgentQuerier, npq querier.AgentNetworkPolicyInfoQuerier
 }
 
 func newConfig(bindPort int) (*genericapiserver.CompletedConfig, error) {
+	os.Remove(TokenPath)
```

Comment:
This is not very readable, you can use `ioutil.WriteFile` when writing file.

Reply:
Good suggestion, Updated.